public/js/home.js

```diff
@@ -131,5 +131,23 @@ $.get('/conversation/search', {
 				$contactsContent.append('<span class="recent-name"><a href="#user-modal" class="recent-contact" data-id="'+ uniqueNames[i].dataset.id +'" rel="leanModal">'+ uniqueNames[i].innerHTML +'</a></span>');
 			}  
 		}
+
+		// Displaying convoshapes
+		var $convoEntry = $('.convo');
+		$convoEntry.each(function(){
+			var $numOfNames = $(this).children('.name').length;
```

Comment:
I understand now. I initially thought the `$` was to be used because of the `$(this)`, but this makes much more sense. Addressed in update.